src/common/cuda/rtc/backward_functions-inl.h

```diff
@@ -53,10 +53,9 @@ backward_log_sigmoid(const DTypeGrad grad, const DType val) {
 template <typename DType, typename DTypeGrad>
 __device__ inline mixed_type<DTypeGrad, DType>
 backward_mish(const DTypeGrad grad, const DType val) {
-  const mixed_type<DTypeGrad, DType> v = val;
-  const auto softrelu = op::log(1 + exp(v));
-  const auto tanh = op::tanh(softrelu);
-  return grad * (tanh + v * sigmoid(v) * (1 - tanh * tanh));
+  const auto softrelu = (val > 20) ? val : op::log(1 + op::exp(val));
```

Comment:
Why not just use `op::softrelu(val)` here instead of replicating that logic?

Reply:
Done